en/02_Developer_Guides/08_Performance/00_Partial_Caching.md

```diff
@@ -62,57 +61,50 @@ otherwise. By using aggregates, we do that like this:
 %>
 ```
 
-The cache for this will update whenever a page is added, removed or edited.
-
-[note]
-The use of the fully qualified classname is necessary.
-[/note]
-
-[note]
-The use of both `.max('LastEdited')` and `.count()` makes sure we check for any object
-edited or deleted since the cache was last built.
-[/note]
-
-[warning]
-Be careful using aggregates. Remember that the database is usually one of the performance bottlenecks.
-Keep in mind that every key of every cached block is recalculated for every template render, regardless of caching
-result. Aggregating SQL queries are usually produce more load on the database than simple select queries,
-especially if you query records by Primary Key or join tables using database indices properly.
-
-Sometimes it may be cheaper to not cache altogether, rather than cache a block using a bunch of heavy aggregating SQL
-queries.
-
-Let us consider two versions:
-
-```ss
-# Version 1 (bad)
-
-<% cached
-    $List('SilverStripe\CMS\Model\SiteTree').max('LastEdited'),
-    $List('SilverStripe\CMS\Model\SiteTree').count()
-%>
-    Parent title is: $Me.Parent.Title
-<% end_cached %>
-```
-
-```ss
-# Version 2 (better performance than Version 1)
-
-Parent title is: $Me.Parent.Title
-```
-
-`Version 1` always generates two heavy aggregating SQL queries for the database on every
-template render.
-`Version 2` always generates a single and more performant SQL query fetching the record by its Primary Key.
+> [!NOTE]
+> The use of the fully qualified classname is necessary.
+>
+> The use of both `.max('LastEdited')` and `.count()` makes sure we check for any object
+> edited or deleted since the cache was last built.
 
-[/warning]
+The cache for this will update whenever a page is added, removed or edited.
 
-[warning]
-If you use the same aggregate in a template more than once, it will be recalculated every time
-unless you move it out into a separate
-[controller method](../templates/partial_template_caching/#cache-key-calculated-in-controller).
-[Object Caching](../templates/caching/#object-caching) only works for single variables and not for chained expressions.
-[/warning]
```

Comment:
Folded this into the other warning to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.